lib/ScannerFactory.php

```diff
@@ -54,6 +54,11 @@ public function __construct(AppConfig $appConfig, ILogger $logger) {
 	 * @throws InitException
 	 */
 	protected function getScannerClass() {
+		$class = $this->appConfig->getExternalScannerClass();
```

Comment:
We might want to include a code comment to ensure we want to prioritize the external scanner over whatever is configured inside the app. We could use another option -> daemon, socket, executable, configured_in_system_config, for example.

Reply:
Config section has been completly disabled

![Screenshot from 2020-11-09 14-42-10](https://user-images.githubusercontent.com/1005065/98548544-f0c95880-2299-11eb-8b4c-8fd781f6c70b.png)